api/api_test.go

```diff
@@ -252,6 +252,25 @@ func TestAdminRedirect(t *testing.T) {
 	}
 }
 
+func TestPostPurchaseIndex(t *testing.T) {
+	api := New(config, apiRoot)
+	response := getHTMLResponse(api, t, secureHost, "/extensions/00000000-0000-0000-0000-000000000002")
+
+	staticContent := "This page is served by your local UI Extension development server. " + 
+	"Instead of visiting this page directly, you will need to connect your local development environment to a real checkout environment."
+	dynamicContent := "If this is the first time you're testing a Post Purchase extension, please install the Chrome or Firefox browser extension from https://github.com/Shopify/post-purchase-devtools/releases. " +
+	"Once installed, simply enter your extension URL /extensions/00000000-0000-0000-0000-000000000002."
```

Comment:
Should this not include the public url? Looks like it just has the relative path?